### PR TITLE
Fit map in corresponding container

### LINKED ISSUE #30: Set Map in RunCard
## User story

Map should be displayed in the RunCard.

### PR DESCRIPTION
**Pull request by Phil9219**
- fit map in big an normal container
- set new routes

### CODE REVIEW

File: client/src/components/Map/Map.js [hunk 1]
```diff
@@ -99,11 +105,11 @@ export default function Map() {
...
 
   return (
-    <div>
-      <MapHeader>
+    <>
+      {/* <MapHeader>
         <Search panTo={panTo} />
         <Locate panTo={panTo} />
-      </MapHeader>
+      </MapHeader> */}
```
Comment (lmachens) [inline client/src/components/Map/Map.js:109]: If you don't need it, consider to remove it :)

File: client/src/components/Page/ChooseYourRunPage.js [hunk 2]
```diff
@@ -28,32 +28,22 @@ export default function ChooseYourRunPage() {
...
-              time={"14:44"}
-              distance={"10"}
-            ></RunCard>
+          <RunCard
+            date={"44.44.444"}
+            time={"14:44"}
+            distance={"10"}
+            isFavorite={true}
+            runName={"Severin Brücke Run"}
+          ></RunCard>
```
Comment (lmachens) [inline client/src/components/Page/ChooseYourRunPage.js]: You can use short form here (multiple times)
```suggestion
          />
```

File: client/src/components/Map/Map.js [hunk 3]
```diff
@@ -49,11 +49,17 @@ const MapHeader = styled.div`
     border: none;
   }
 `;
+const InfoBubble = styled.div`
+  color: var(--primary-color);
+  .p {
+    color: var(--primary-color);
+  }
```
Comment (lmachens) [inline client/src/components/Map/Map.js]: I think you can remove this
```suggestion
```

**Review by lmachens — COMMENTED** [review]
good job

**Review by SvLowman — APPROVED** [review]
👍

File: client/src/components/Map/Map.js (hunk 1 above)
Comment (Phil9219) [reply client/src/components/Map/Map.js:109]: I´m still working on it. I want to ```Search panTo``` and ```Locate panTo```to be integrated in the map.

File: client/src/components/Page/ChooseYourRunPage.js (hunk 2 above)
Comment (Phil9219) [reply client/src/components/Page/ChooseYourRunPage.js]: Thanks

File: client/src/components/Map/Map.js (hunk 3 above)
Comment (Phil9219) [reply client/src/components/Map/Map.js]: Thanks

File: client/src/components/Map/Map.js (hunk 1 above)
Comment (Phil9219) [reply client/src/components/Map/Map.js:109]: Opened a new issue for it an will merge this branch

**Pull request merged**